tools/ci_build/build.py

```diff
@@ -1861,6 +1861,10 @@ def setup_rocm_build(args):
 
 
 def run_android_tests(args, source_dir, build_dir, config, cwd):
+    if args.android_abi != "x86_64":
+        log.info(f"--android_abi ({args.android_abi}) is not x86_64, skipping running of Android tests on emulator.")
+        return
+
```

Comment:
Is this too restrictive now that there are more arm64 based desktop/laptops which assumably the android emulator could be run on?

Reply:
yes. I think it's only excluding arm64 mac environments now. it says here that ARM CPUs for Windows/Linux aren't currently supported.
https://developer.android.com/studio/install

this logic wasn't actually changed in this PR, just added some logging. I think we can relax this if needed later.